public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -430,9 +430,9 @@ async function Sidebar() {
 </div>`,
 			"usage": `<div id="sb4"  class="custom_sidebar">
  
-    <!--<div class="custom_field">
+    <div class="custom_field">
          <a onclick = "event.preventDefault(); buildcite()" href='#' class='mt-icon-quote'>&nbsp;Get Page Citation</a>
```

Comment:
```suggestion
         <a onclick = "event.preventDefault(); buildcite()" href='#' class='mt-icon-quote'>Get Page Citation</a>
```

Reply:
We need the white space as otherwise the icon is too close to the text